src/test/fixtures/profiles/tracks.js

```diff
@@ -172,6 +172,36 @@ export function getProfileWithNiceTracks(): Profile {
   return profile;
 }
 
+/*
+ * This produces profile where no global track will be found. This is simulating
+ * a Chrome importer profile.
```

Comment:
I think this is not 100% accurate as this profile wasn't from a chrome importer. We can just say "other importer profiles" probably.

Reply:
Yeah, it was a wrong assumption, I understood that later.